src/app/navbar/navbar.component.html

```diff
@@ -14,31 +14,31 @@
         <!--<li><a routerLink="/advertisements" [routerLinkActive]="'active'" [routerLinkActiveOptions]="{ exact: true }">Advertisements</a></li>-->
         <!--<li><a routerLink="/searchAdvertisements" [routerLinkActive]="'active'" [routerLinkActiveOptions]="{ exact: true }"> Search Advertisements</a></li>-->
         <!-- <li><a href="#contact">Contact</a></li> // move to footer -->
-        <app-search-ad></app-search-ad>
+        <!--<app-search-ad></app-search-ad>-->
```

Comment:
@SrBlimp For some reason search component on navbar was commented, even though it was working correctly. It is now uncommented.